pahfit/PAHFITBase.py

```diff
@@ -225,12 +229,180 @@ def plot(self, ax, x, y, model):
         # print(obs_fit.param_names)
         # print(obs_fit.parameters)
 
-    def save(self, filename):
+    def save(self, obs_fit, filename, outform):
```

Comment:
Needs an updated docstring (info between """ and """).

Reply:
Yes. How do we proceed with this? Update the docstring and then make a new pull request?

Reply:
Just update the docstring and push to the same branch.  The pull request will automatically update.  :-)

Reply:
Cool! I'll do that.